### ISSUE TITLE
'#' in database name cause a bug in web ui

### ISSUE THREAD

**Issue by GaoYusong** [influxdb/influxdb#725]
i created two dbs named by warden#hongkong and warden from web ui, it is success.

but i cann't insert record into warden#hongkong,

even more when i delete warden#hongkong, the server will delete warden not warden#hongkong.

i install influxdb-0.7.3 from brew.

thanks a lot

**Comment by otoolep** [comment]
This has been fixed in 0.9.0. 
```
$ curl -G 'http://localhost:8086/query' \
> --data-urlencode "q=CREATE DATABASE mydb#"
{"error":"error parsing query: found #, expected ;, EOF at line 1, char 21"}
```
It's no longer possible to create a database with a `#` in the name.

**Comment by GaoYusong** [comment]
Thanks a lot for your great work!
